src/OpenTelemetry.Instrumentation.SqlClient/SqlClientTraceInstrumentationOptions.cs

```diff
@@ -278,12 +305,11 @@ internal void AddConnectionLevelDetailsToActivity(string dataSource, Activity sq
             }
             else
             {
-                sqlActivity.SetTag(SemanticConventions.AttributeServerSocketAddress, connectionDetails.ServerIpAddress);
+                sqlActivity.SetTag(SemanticConventions.AttributeServerAddress, connectionDetails.ServerIpAddress);
             }
 
             if (!string.IsNullOrEmpty(connectionDetails.Port))
             {
-                // TODO: Should we continue to emit this if the default port (1433) is being used?
                 sqlActivity.SetTag(SemanticConventions.AttributeServerPort, connectionDetails.Port);
```

Comment:
Removed this TODO because the instrumentation already excludes port when it is 1433.